### ISSUE TITLE
missing headers (using Fastify)

### ISSUE THREAD

**Issue by kireerik** [enisdenjo/graphql-sse#91]
We are using `@fastify/` `cors` and `cookies` and the headers are missing when using this library.

If I change this line (https://github.com/enisdenjo/graphql-sse/blob/061b0fc48b3b2a892fde39de2dac3c67f5623d44/src/use/fastify.ts#L90) like this (
```Node.js
    reply.raw.writeHead(init.status, init.statusText, {...reply.getHeaders(), ...init.headers});
```
) then the `cors` headers are not missing, but the `cookie` header is still missing.

**Comment by enisdenjo** [comment]
Hey there, the `Cookie` header does not come from the server - it comes only from  the client ([see MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cookie)).

However, a server can send a `Set-Cookie` header though. Can you try that?

Regarding the rest of the headers, a PR is very welcome - would you consider opening one? Thanks in advance!

**Comment by kireerik** [comment]
Yes, that's what is not working as well. We are using `@fastify/cookie` to set a cookie from the server side using `Set-Cookie`.

Sure, I can for sure create a pull request for the `cors` case. However the missing `Set-Cookie` header is a bit unclear for me. I am not sure how should I get it. We set the cookie from a (GraphQL) (mutation) resolver (using `@fastify/cookie`).

**Comment by enisdenjo** [comment]
Is the set-cookie header present without graphql-sse? Maybe it is not in the `getHeaders` (you can check that by console logging the results)?

**Comment by kireerik** [comment]
Yes, it is. Yes it is not in the `getHeaders`.

Anyway. Now we are using the GraphQL Yoga server which (after resolving 2 issues related to it) seems to work as expected.

**Comment by enisdenjo** [comment]
Sorry for waiting so long! This issue has been fixed with https://github.com/enisdenjo/graphql-sse/commit/134c1b022599e5c403d7204eda61106bac34216a and released in [v2.5.3](https://github.com/enisdenjo/graphql-sse/releases/tag/v2.5.3). Please check it out, thanks!
